Ascend.java

```diff
@@ -2,6 +2,13 @@
 // and prints them in increasing order.
 public class Ascend {

```

Comment:
OVERALL: 
1. A very good assignment. there are 2 main concerns in logical aspect, one in Ascend.java and the other in TimeFormat.java
2. Styling wise it was very good (aside for few mistakes)
3. Commits names should be better